### DISCUSSION TITLE
Does Orgro have snippets for easier writing on mobile?

### DISCUSSION THREAD

**Discussion by kresdjan** [amake/orgro#127] [General]
Hi, I am using Orgro on iOS. Does Orgro add support for writing similar to org-mode, where `|+` expands into a table?

**Comment by amake** [comment]
Orgro offers some writing convenience features in the editor:
﻿﻿
![IMG_981C4663AFC7-1](https://github.com/user-attachments/assets/0bd9e2b1-93bd-482c-a7e0-cbb2a25631c7)

(Note that you can slide the toolbar to reveal more items on the right.)

I currently don't offer anything specific for tables.

If I was going to offer something for tables, I'm not sure I would want it to work like it does in org-mode. The org-mode editing affordances assume you are typing on a physical keyboard, where it's easy to e.g. press Tab to "expand" something. On a touchscreen I think a more tactile, GUI-based affordance would be more appropriate. That could look like some sort of table creation dialog box triggered by an editor toolbar item, or, even better, some sort of table-editing controls in the regular view.
